python/ray/tune/result.py

```diff
@@ -1,4 +1,9 @@
 import os
+from ray.air.constants import (
+    TIMESTAMP,
+    TIME_THIS_ITER_S,
+    TRAINING_ITERATION,
+)
```

Comment:
Why do we need to import these again? Is it to preserve existing imports from `ray.tune.result`?

Reply:
There are other constant lists, such as `AUTO_RESULT_KEYS`, that still use these three constants. Once we migrate them together with the modules, there will be no need to import them again.